### PR TITLE
Send Logs to a Location That Musher can Write To

### PR DESCRIPTION
**Pull request by bencap**

### CODE REVIEW

**Comment by bencap** [comment]
[Turn on structured logging in husky musher](https://trello.com/c/IuFLAQFj/1170-turn-on-structured-logging-in-husky-musher)

**Comment by bencap** [comment]
Will also need to add logrotate to this directory, maybe will also throw up some logrotate documentation

File: logging.yaml [hunk 1]
```diff
@@ -1,19 +1,12 @@
 ---
 version: 1
 
-# Leave the new structured log handler turned off until we figure out log storage destination.
-# Currently written as a file handler for structured logging proof of concept.
-
-# root:
-#   handlers:
-#     - fileHandler
```
Comment (kschwabefry) [inline logging.yaml:9]: Do we still need this part of the config?

Comment (bencap) [reply logging.yaml:9]: Yup totally, my bad - dropped this on accident while rebasing after the reversion.

**Review by kschwabefry — APPROVED** [review]

**Comment by bencap** [comment]
Deployed.

**Comment by bencap** [comment]
Added log location to logrotate.

**Pull request merged**